src/rotations.jl

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    DatamineAngles(α, β, θ)
+
+Datamine ZXY rotation convention following the left-hand rule.
+All angles are in degrees and the signal convention is CW, CW, CW
+positive. Y is the principal axis.
+"""
+
```

Comment:
We can't add a white line between the docstring and the struct otherwise Julia won't connect the two.